digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/util/AssetDataImporter.scala

```diff
@@ -62,12 +57,19 @@ AssetDataImporter {
   }
 }
 
+protected case class RoadAddressHistory(roadNumber: Long, roadPartNumber: Long, trackCode: Long, discontinuity: Long,
+                                        startAddrM: Long, endAddrM: Long, startM: Double, endM: Double, startDate: Option[DateTime], endDate: Option[DateTime],
+                                        validFrom: Option[DateTime], validTo: Option[DateTime], ely: Long, roadType: Long,
+                                        terminated: Long, linkId: Long, userId: String, x1: Option[Double], y1: Option[Double],
+                                        x2: Option[Double], y2: Option[Double], lrmId: Long, ajrId: Long)
+
 class AssetDataImporter {
   val logger = LoggerFactory.getLogger(getClass)
   lazy val ds: DataSource = initDataSource
 
   val Modifier = "dr1conversion"
-  val dateFormatter = ISODateTimeFormat.basicDate()
+
+  lazy val roadMaintainerElys = Seq(0, 1, 2, 3, 4, 8, 9, 10, 12, 14)
```

Comment:
This is no longer needed, because we don't run the batch import by "ely" anymore